docs/reference/configuration/istio-sidecar-vs-ambient.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Istio Sidecar vs. Ambient Mode in UDS Core
```

Comment:
Okay with this as-is to add the content but we may want to add a sub-folder for "Service Mesh" (or explicitly istio) and put this doc + the istio ingress docs under it?

Reply:
One thing I should've noted - when we move docs we'll also want to add redirects to the uds-docs site so that the previous links resolve to the new location.

Two other docs to potentially add to this folder:
- https://github.com/defenseunicorns/uds-core/blob/docs/ambient/docs/reference/configuration/authorization-policies.md
- https://github.com/defenseunicorns/uds-core/blob/docs/ambient/docs/reference/configuration/non-http-ingress.md